uqcsbot/scripts/yt.py

```diff
@@ -0,0 +1,55 @@
+import os
+from uqcsbot import bot, Command
+
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+
+YOUTUBE_API_KEY = os.environ.get('YOUTUBE_API_KEY')
+YOUTUBE_API_SERVICE_NAME = 'youtube'
+YOUTUBE_API_VERSION = 'v3'
+YOUTUBE_VIDEO_URL = 'https://www.youtube.com/watch?v='
+
+
+@bot.on_command('yt')
+def handle_yt(command: Command):
+    '''
+    `!yt <QUERY>` - Returns the top video search result based on the query string.
+    '''
+    # Makes sure the query is not empty.
+    if not command.has_arg():
+        bot.post_message(command.channel, "You can't look for nothing. !yt <QUERY>")
```

Comment:
pls make this message a constant outside the function so testing can point at it directly